edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/ClientConnectionHandler.cs

```diff
@@ -47,7 +47,8 @@ public Task<IDeviceListener> GetDeviceListener()
                                 .GetOrElse(
                                     async () =>
                                     {
-                                        IDeviceListener dl = await this.connectionProvider.GetDeviceListenerAsync(this.identity);
+                                        // TODO: Implement plug and play for AMQP
```

Comment:
> // TODO: Implement plug and play for AMQP [](start = 39, length = 42)

Put the task/PBI ID, otherwise if plans changes then chances are we won't clean up this stub.

Reply:
Normally we don't want to put internal links in iotedge repo as it is an open source project.